CHANGELOG.md

```diff
@@ -8,7 +8,15 @@
 
 ##### Bug Fixes
 
-* None.  
+* Correctly handle `--help` flags when using `argv.remainder!` after initialization  
+  [Eric Amorde](https://github.com/amorde),
+  [tripleCC](https://github.com/tripleCC)
+  [#87](https://github.com/CocoaPods/CLAide/pull/87)
+
+
```

Comment:
nit empty line

Reply:
are you saying there's an extra line? I copied the style from the other sections